app/auto_responder_test.go

```diff
@@ -197,6 +197,43 @@ func TestSendAutoResponseIfNecessary(t *testing.T) {
 		assert.Nil(t, err)
 		assert.False(t, sent)
 	})
+
+	t.Run("should send auto response in dm channel if not already sent today", func(t *testing.T) {
+		th := Setup(t).InitBasic()
+		defer th.TearDown()
+
+		receiver := th.CreateUser()
+
+		patch := &model.UserPatch{
+			NotifyProps: map[string]string{
+				"auto_responder_active":  "true",
+				"auto_responder_message": "Hello, I'm unavailable today.",
+			},
+		}
+		receiver, err := th.App.PatchUser(receiver.Id, patch, true)
+		require.Nil(t, err)
+
+		channel := th.CreateDmChannel(receiver)
+
+		savedPost, err := th.App.CreatePost(&model.Post{
+			ChannelId: channel.Id,
+			Message:   "zz" + NewTestId() + "a",
```

Comment:
The objective of using the testID is to get rid of the `zz` and `a` hacks. We can remove them now.